logging/bin/change_internal_password.sh

```diff
@@ -155,7 +155,8 @@ if [[ $response == 4* ]]; then
       # make sure hash utility is executable
       kubectl -n $LOG_NS exec $targetpod  -c $targetcontainer --  chmod +x $toolsrootdir/tools/hash.sh
       # get hash of new password
-      hashed_passwd=$(kubectl -n $LOG_NS exec $targetpod  -c $targetcontainer --  $toolsrootdir/tools/hash.sh -p $NEW_PASSWD)
```

Comment:
Several months back, we had to add the grep command to a similar/identical command (line 117) b/c OpenSearch decided to add a deprecation notice in the output of the hash.sh script.  We should have done it here as well and didn't.  Correcting that oversight now.